onnxscript/tests/function_libs/torch_lib/ops_test_data.py

```diff
@@ -573,7 +573,7 @@ def _where_input_wrangler(
     "scatter_reduce": (core_ops.aten_scatter_reduce, _scatter_reduce_input_wrangler),
     "slice_scatter": core_ops.aten_slice_scatter,
     "slice": core_ops.aten_slice,
-    "stft": core_ops.aten_stft,
+    "aten_stft": core_ops.aten_stft,
```

Comment:
```suggestion
    "aten.stft": core_ops.aten_stft,
```

Maybe this to separate it from the rest?